pkg/scheduler/framework/types_test.go

```diff
@@ -214,7 +215,7 @@ type testingMode interface {
 	Fatalf(format string, args ...interface{})
 }
 
-func makeBasePod(t testingMode, nodeName, objName, cpu, mem, extended string, ports []v1.ContainerPort, volumes []v1.Volume) *v1.Pod {
+func makePod(t testingMode, nodeName, objName, cpu, mem, extended string, ports []v1.ContainerPort, volumes []v1.Volume) *v1.Pod {
```

Comment:
Can we use `st.MakePod()` directly instead of adding another layer? This is the starting point of the issue.

Reply:
are you suggesting calling ``st.MakePod()`` in every test function instead of using makePod() ? and construct it base on every function?

Reply:
but ``makePod`` have params like ``cpu, mem, extended string`` which ``st.MakePod`` don't have. and we have some if logic in ``makePod``. I think if we use ``st.MakePod`` directly, then we need to wrapped the cpu, mem, extened in pod_wrapper. Please correct me if I'm wrong

Reply:
Actually I think you can, for instance:
```golang
		st.MakePod().UID("test-2").Namespace("node_info_cache_test").Name("test-2").Node(nodeName).Containers([]v1.Container{st.MakeContainer().ResourceRequests(map[v1.ResourceName]string{
			v1.ResourceCPU:    "200m",
			v1.ResourceMemory: "1Ki",
		}).ContainerPort([]v1.ContainerPort{v1.ContainerPort{
			HostIP:   "127.0.0.1",
			HostPort: 8080,
			Protocol: "TCP",
		}}).Obj()}).Obj()
```

Reply:
I've addressed all the comments above. PTAL when you get a chance. Thanks! @kerthcet